web/client/src/components/request/RequestModule.jsx

```diff
@@ -17,18 +17,38 @@ import {
   DialogContent,
   FormHelperText,
   IconButton,
+  Collapse,
 } from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
 import AddCircleOutlineIcon from "@material-ui/icons/AddCircleOutline";
 import DeleteIcon from "@material-ui/icons/Delete";
+import CloseIcon from "@material-ui/icons/Close";
 import "../../styles/request.css";
 import { object, string } from "yup";
+import { getHTTPResponse } from "../../util/apiWrapper";
+import prettifyJSON from "../../util/prettify_json";
 
 /**
  * @return {Formik} component using Formik for creating send Request
  */
 export default function RequestModule() {
   const [openHeaderBox, setOpenHeaderBox] = useState(false);
   const [openRequestBodyBox, setOpenRequestBodyBox] = useState(false);
+  const [response, setResponse] = useState("");
+  const [errorOpen, setErrorOpen] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const [errMessage, setErrMessage] = useState("");
+
+  const getResponse = async (values) => {
+    const Response = await getHTTPResponse(values);
```

Comment:
Lower case var

Reply:
done! Fixed also for other files as well